pkg/operator/ceph/pool/controller.go

```diff
@@ -165,32 +171,53 @@ func (r *ReconcileCephBlockPool) reconcile(request reconcile.Request) (reconcile
 	// Populate clusterInfo during each reconcile
 	clusterInfo, _, _, err := mon.LoadClusterInfo(r.context, request.NamespacedName.Namespace)
 	if err != nil {
-		return reconcile.Result{}, errors.Wrap(err, "failed to populate cluster info")
+		return opcontroller.ImmediateRetryResult, errors.Wrap(err, "failed to populate cluster info")
```

Comment:
What is the difference in retry if we return `reconcile.Result{}` compared to `ImmediateRetryResult`? Looks like in other controllers we are mostly returning the first one. 

I just want to make sure we aren't increasing the retry rate after the operator is restarted. It can take a minute or two before the config is ready for the controllers to consume. For a separate work items I'd actually like to clean up these errors that appear in the operator log while they're waiting for that initialization.

Reply:
My mocks were only behaving correctly when enforcing the immediate return, which is what the controller does anyways...
But that's only with DEBUG logging right?

Reply:
I'll open a new issue with the details when I see it.